config.py

```diff
@@ -11,6 +11,9 @@ class Config:
     )
     AUTH_REQUIRED = True
     ALLOW_EXPLORER = True
+
+    ELASTICSEARCH_HOST = os.getenv('DM_ELASTICSEARCH_URL', 'localhost:9200')
```

Comment:
Does having this default here mean that we can remove the `DM_ELASTICSEARCH_URL` from the run scripts?